playbooks/bootstrap_ml2_vlan.yml

```diff
@@ -36,6 +36,11 @@
     - ucarp
   - apt: upgrade=dist
 
+  - name: load vlan module
+    lineinfile: dest=/etc/modules
+                line="8021q"
+    when: ansible_distribution_version == "14.04"
```

Comment:
the module wasn't listed until I added this and I was having issues getting vlans to work until I added it and rebooted.   Docs say to do it.